kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -154,11 +154,11 @@ impl<'tcx> GotocCtx<'tcx> {
         if self.queries.get_check_assertion_reachability() {
             // Generate a unique ID for the assert
             let assert_id = self.next_check_id();
-            // Generate a message for the reachability check that includes the unique ID
-            let reach_msg = assert_id.clone();
             // Also add the unique ID as a prefix to the assert message so that it can be
             // easily paired with the reachability check
             let msg = GotocCtx::add_prefix_to_msg(&msg, &assert_id);
+            // Generate a message for the reachability check that includes the unique ID
+            let reach_msg = assert_id;
```

Comment:
Hmm, yes, getting rid of a `.clone()` is nice, but seems unrelated to toolchain updates?

Reply:
I agree that it's not ideal, but also not worth the review cycle to remove these trivial changes, don't you think? Anyway, I'll revert these for now.

Reply:
I do not at all intend to block this PR over this, it's just (like the whitespace change) something that tripped me up while reading the changes. Might be a matter of personal preference that I'd rather do 5 PRs that each do one thing.